### ISSUE TITLE
Remove aria-labelledby which is not needed anymore

### ISSUE THREAD

**Issue by ngarnier** [mjmlio/mjml#419]
Following [Gmail's changes regarding responsive](https://www.mailjet.com/blog/news/gmail-responsive-email-design), `aria-labelledby` should not be needed anymore and should be removed without causing any issue.
